book/src/installers/other.md

```diff
@@ -1,13 +1,13 @@
 # Other Installation Methods
 
-cargo-dist projects can also theoretically be installed with the following, through no active effort of our own:
+dist projects can also theoretically be installed with the following, through no active effort of our own:
 
 * [cargo-install][] (just [cargo publish][] like normal)
```

Comment:
this is probably a more general note but talking about cargo install here doesn't make much sense anymore

Reply:
We can probably get rid of this whole section? It doesn't seem especially relevant even to our Rust users.